src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -1233,6 +1233,14 @@ public PrivateChannel createPrivateChannel(DataObject json)
         if (channel != null)
             return channel;
 
+        //if we don't have enough information to construct the channel properly, the rest of the properties can be null
+        if (!json.hasKey("recipients") && !json.hasKey("recipient"))
```

Comment:
Above this, we return without checking if the channel already has a user attached, maybe we should add a check and populate the user there if provided?